drivers/ieee802154/Kconfig

```diff
@@ -103,6 +103,16 @@ config IEEE802154_SELECTIVE_TXPOWER
 	help
 	  Enable support for selectively setting TX power for every transmission request.
 
+config IEEE802154_MULTIPLE_CCA
+	bool "Support for multiple CCA attempts before transmission"
+	help
+	  This is an optional extension not conforming to IEEE Std. 802.15.4-2015.
+	  When this option is enabled the user of ieee802154_radio_api has possibility
+	  to express the maximum number of extra CCA attempts for net_pkt. The extra
+	  CCA attempts are performed back-to-back when previous CCA attempt ended with
+	  a busy channel condition. Beause of that the moment of transmission can be
```

Comment:
typo: `Because`. Also, maybe it's worth explicitly stating that the CCA attempts are performed until either idle channel is detected or the maximum number provided by the caller is reached.